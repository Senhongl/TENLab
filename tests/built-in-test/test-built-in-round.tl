a = [1,2,4];
print(round(a));
b = [0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8,0.9,1.0];
print(round(b));
