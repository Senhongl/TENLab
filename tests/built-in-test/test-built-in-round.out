 1
 2
 4
[ CPUIntType{3} ]
 0
 0
 0
 0
 0
 1
 1
 1
 1
 1
[ CPUDoubleType{10} ]